src/CRM/CivixBundle/Resources/views/Code/module.php.php

```diff
@@ -12,7 +16,7 @@
  *
  * @link https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_config/
  */
-function <?php echo $mainFile ?>_civicrm_config(&$config): void {
+function <?php echo $mainFile ?>_civicrm_config(\CRM_Core_Config $config): void {
```

Comment:
Tangential: It's weird that the hook defines `$config` as pass-by-reference. The `$config` is so big and central... it's hard for me to imagine what happens if one replaces that object. (My gut says that part of the of hook signature is leftover from the PHP4/PHP5 era, with the rationale lost to the sands of time...)

In any case, it's probably good that this makes it _look_ like a regular object-param. Inviting people to think of it as a reference is... inviting trouble (AFAICS, for no real reason)...

Aside: We should also add the newer `?array $flags = NULL` because anyone who actually uses this hook probably should use `$flags`. (But that's can be a separate issue.)